packages/cxl-ui/src/components/cxl-course-modal.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import '@vaadin/button';
+import '@vaadin/dialog';
+import './cxl-time.js';
+import './jw-player/index.js';
+import { dialogFooterRenderer, dialogRenderer } from '@vaadin/dialog/lit.js';
+import cxlCourseModalGlobalStyles from '../styles/global/cxl-course-modal-css.js';
+
+@customElement('cxl-course-modal')
```

Comment:
1. My mistake, I think when reading about dialogs I must have copied the tag from the source code. <vaadin-dialog> seems also correct to me.

Reply:
> Isn't it more beneficial to extend <vaadin-dialog> instead of creating custom component from scratch? I see opened state being managed here. Isn't it already done in Vaadin component?

I don't see anything wrong with using a wrapper. Especially if we want the open button to be part of the component.